Assets/Scripts/Weapons/Gun.cs

```diff
@@ -33,7 +33,11 @@ public void Awake()
     public void Shoot(bool firstShot = false)
     {
         if (GameManager.Instance.IsPaused) return;
-        if ( !readyToShoot || WeaponManager.Instance.BulletsInMag <= 0 || reloading) return;
+        if ( !readyToShoot || WeaponManager.Instance.BulletsInMag <= 0 || reloading)
+            return;
+        else
+            AudioManager.Instance.Play(GunSO.name);
```

Comment:
This should be further down - the audio will still play if the weapon wheel is open. I suggest placing the call after line 73 `OnGunShootEvent?.Invoke();`